src/main/java/core/basesyntax/service/impl/DataConvertorImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConvertor;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConvertorImpl implements DataConvertor {
+    public static final int MINIMUM_SIZE = 2;
+    public static final String CHAR_FOR_SPLIT = ",";
```

Comment:
you can simplify name
```suggestion
    private static final String COMMA = ",";
```